tfidf-benchmark/NLP-pipeline-spark.ipynb

```diff
@@ -0,0 +1,673 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #36.                        featurizedData.head(0)
</pre><p>Same here. I suggest just benchmarking till this bit with head(0) . </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/rapids-examples/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>You mean removing the IDF part? I can persist the <code>featurizedData</code>  so that it performs a compute there. But I do not think in a tf-idf pipeline it makes sense to stop at <code>hashingTF</code>  . The latencies of different parts of the pipeline are compartmentalized anyway. We can choose not to use the <code>hashingTF</code>  one even though it is in the complete pipeline. </p>